lib/preamble.dart

```diff
@@ -124,7 +125,7 @@ if (dartNodeIsActuallyNode) {
 ///
 /// If [minified] is true, returns the minified version rather than the
 /// human-readable version.
-String getPreamble({bool minified: false, List<String> additionalGlobals: const []}) =>
+String getPreamble(
+        {bool minified: false, List<String> additionalGlobals: const []}) =>
     (minified ? _minified : _normal) +
-    (additionalGlobals == null ? "" :
-        additionalGlobals.map((global) => "self.$global=$global;").join());
+    (additionalGlobals.map((global) => "self.$global=$global;").join());
```

Comment:
I believe these changes need to be made to `tool/minify.js`. This is generated code.

Reply:
oh, ok I am was just naively doing things here I can follow up (but will need to wait to tomorrow)

Reply:
I see now that I probably need to update the actual generated code as well?

Reply:
@jakemac53 maybe I don't understand, sorry. the `final _minified` here is generated by `tool/minified.js`. `lib/preamble.dart` is generated by `tool/minified.js`.

Reply:
Ya no I figured it out :)